server/package.json

```diff
@@ -59,6 +62,7 @@
     "mongodb-memory-server": "^7.4.3",
     "mongoose": "^6.0.8",
     "node-cache": "^5.1.2",
+    "node-fetch": "2",
```

Comment:
Is this to avoid the ESM-only node-fetch 3 upgrade?

Reply:
New commit should revert it back to ESM only!